README.md

```diff
@@ -23,29 +24,39 @@ Setup
 
 pa11y-dashboard requires [Node.js][node] 0.10+ and [PhantomJS][phantom]. See the [pa11y][pa11y] documentation for detailed instructions on how to install these dependencies on your operating system.
 
-You'll also need to have [MongoDB][mongo] installed and running. See the [MongoDB install guide][mongo-install] for more information on this. 
+You'll also need to have [MongoDB][mongo] installed and running. See the [MongoDB install guide][mongo-install] for more information on this.
 
-You'll then need to clone this repo locally and install dependencies with `npm install`.
+You'll then need to clone this repo locally and install dependencies with `npm install`. Now we need to add some configuration before we can run the application. We can do this in two ways:
```

Comment:
Can we stick with "you", rather than switching to "we"? Do you think that would read a little better?